Dockerfile

```
@@ -8,10 +8,13 @@ RUN echo "deb http://packages.cloud.google.com/apt cloud-sdk-jessie main" | tee
       echo "deb http://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
       curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
       curl -sL https://deb.nodesource.com/setup_11.x | bash - && \
+      wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - && \
+      sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' && \
       apt-get update && \
-      apt-get install --no-install-recommends -y google-cloud-sdk nodejs yarn && \
+      apt-get install -y gcc python-dev python-pip python-setuptools google-cloud-sdk nodejs yarn google-chrome-unstable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst ttf-freefont --no-install-recommends && \
```

Let's kill the fonts, slightly less weight
```suggestion
      apt-get install -y gcc python-dev python-pip python-setuptools google-cloud-sdk nodejs yarn google-chrome-unstable --no-install-recommends && \
```